src/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>leaderboard</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65"
+      crossorigin="anonymous"
+    />
+</head>
+<body>
+    <div class="container">
+        <h1>Leaderboard</h1>
+        <div class="row">
+          <div class="col">
+            <h3>
+              Recent scores
+              <button id="refresh" type="button" class="btn btn-dark">
+                Refresh
+              </button>
+            </h3>
+            <table class="table table-striped">
+              <tbody>
+                <tr>
+                  <td>Name: 100</td>
+                </tr>
+                <tr>
+                  <td>Name: 20</td>
+                </tr>
+                <tr>
+                  <td>Name: 50</td>
+                </tr>
+                <tr>
+                  <td>Name: 78</td>
+                </tr>
+                <tr>
+                  <td>Name: 125</td>
+                </tr>
+                <tr>
+                  <td>Name: 77</td>
+                </tr>
+                <tr>
+                  <td>Name: 42</td>
+                </tr>

```

Comment:
- Hey, Although you were not told to display the content using javascript since this is a javascript project, you should display this content dynamically by storing the scores in an array. Also, according to the project requirement, you were asked to use `Use ES6 modules, with import and export.`, so you should know you need to separate your files, etc. Please, kindly do this to fix this issue. :pray: